### PR TITLE
Ques

### PR DESCRIPTION
**Pull request by muskan-gupta-0803**
Questions page added.

### CODE REVIEW

**Review by abhisheksahani-nova — APPROVED** [review]
Looks good to go

File: css/ques.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+.main-container{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.hd{
+    position: fixed;
```
Comment (xsaurabhsuthar) [inline css/ques.css:16]: If you're already using flew then you should avoid using fixed positioning. Better make main area height 100% with flex and use overflow-y: auto. Also make body height 100vh

**Review by arulk8 — APPROVED** [review]

**Review by Balakumar3018 — APPROVED** [review]
Everything looks good. Avoid spaces between lines

**Review by sashaboi — APPROVED** [review]
Looks good to go!

File: css/ques.css (hunk 1 above)
Comment (xsaurabhsuthar) [reply css/ques.css:16]: image

**Review by xsaurabhsuthar — APPROVED** [review]

File: css/ques.css (hunk 1 above)
Comment (muskan-gupta-0803) [reply css/ques.css:16]: thanks for this detailed explanation, i will try to implement this.

**Comment by TJain02** [comment]
The code is clean. Good to go!

**Pull request merged**